compiler/luci/pass/src/QuantizeBias.cpp

```diff
@@ -201,6 +201,12 @@ CircleConst *QuantizeBias::quantized_bias(CircleNode *input, const CircleNode *w
     std::vector<float> scaling_factor(size);
     std::vector<int64_t> zp(size);
 
+    if (size != const_bias->dim(const_bias->rank() - 1).value())
```

Comment:
Just curious... in almost all cases, there are no case for `const_bias->rank() == 0`, am I right?
(i.e. bias is scalar)